src/shared/components/Court/Court.tsx

```diff
@@ -0,0 +1,6 @@
+import { Wrapper } from './styles';
+import { CourtProps } from './types';
+
+export function Court({ children }: CourtProps) {
+  return <Wrapper>{children}</Wrapper>;
```

Comment:
**Question:** Is it possible to use the `Card` component?

Reply:
Let me know if you like this: https://github.com/Wendlereis/minha-vez/pull/32/commits/e2e539ce3265c68c1e169b0efa3d6892e8276f58

Reply:
Cool!